main/settings.py

```diff
@@ -123,18 +122,17 @@
 AUTH_USER_MODEL = "users.User"
 
 # Application definition
-INSTALLED_APPS = (
+INSTALLED_APPS = [
     "django.contrib.admin",
     "django.contrib.auth",
     "django.contrib.contenttypes",
     "django.contrib.sessions",
     "django.contrib.messages",
     "django.contrib.staticfiles",
     "django.contrib.sites",
-    "compat",
     "guardian",
     "hijack",
-    "hijack_admin",
+    "hijack.contrib.admin",
```

Comment:
Not seeing the "hijack" option for users in django admin.  Not sure why, all the pieces seem to be in place.